### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GoorLavi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,49 +21,67 @@ Currently we have copyFiles and copyItemsSync (both having the same parameters)
...
-
-Type: String
-
-Full path to destination location 
+```js
+import {copyFiles} from 'copy-files-tree'
 
+const FoldersData = {
+    '/Users/user/folder name 1': {
+        foldersAndFiles: ['image.png', 'inner folder name1'],
```
Comment (yogevmalcaym) [inline README.md]: I think that the folder names should be inner-seperated with '-'.
like this 'inner-foldername-1'

Comment (yogevmalcaym) [reply README.md]: Same in the whole page

File: README.md [hunk 2]
```diff
@@ -21,49 +21,67 @@ Currently we have copyFiles and copyItemsSync (both having the same parameters)
...
+    },
+    '/Users/user1/folder name 2': {
+        options: {
+            allFiles: true,  // Folder files [`file name1.txt`, `file name2.txt`]
+        },
+        destination: '/Users/user2/some folder name2'
+    }
+}
 
+const error = copyFiles(FoldersData);
```
Comment (yogevmalcaym) [inline README.md]: The error variable will contain Promise.. It won't be more precised if we'll present it like this?
```
copyFiles(FoldersData).then(err=>
  if(err) console.log(err);
)
```
Or something like this:
```
(async function (){
   const error = await copyFiles(FoldersData);
   if(error) 
   console.log(error);
}())
```

File: README.md [hunk 3]
```diff
@@ -21,49 +21,67 @@ Currently we have copyFiles and copyItemsSync (both having the same parameters)
...
 
-const error = copyFiles({
-    FolderPath1: FolderProps,
-    FolderPath2: { allFiles: true, destination: 'FullDestinationFolderPath'},
-    FolderPath3: { allDirectories: true, destination: 'FullDestinationFolderPath'}
-});
+const error = copyFiles(FoldersData);
 ```
 
 ## Props
```
Comment (yogevmalcaym) [inline README.md]: I think you deleted the Props from the last commit.
Btw it may be confusing that here we call it 'Props' and in the library 'Options'

Comment (yogevmalcaym) [reply README.md]: I just saw you wrote 'Options' beneath

Comment (yogevmalcaym) [inline README.md:38]: You can add 'required' and 'optional' references to the props

File: README.md [hunk 4]
```diff
@@ -21,49 +21,67 @@ Currently we have copyFiles and copyItemsSync (both having the same parameters)
...
-#### Result
 
-```
-Destination/wantedFolder/ ------ image.png          
-                               |       
-                               - folder1/ ------  innerFolder/ ----- file.text         
-                               |         
-                               - folder2/ ------  file.txt       
+```js
+/Users/ ----- |
```
Comment (yogevmalcaym) [inline README.md:84]: According to your example the files will be copied into 'some folder name' and 'some folder name1' but in this chart you pointed them into 'user1' and 'user2'

File: README.md (hunk 2 above)
Comment (GoorLavi) [reply README.md]: Oh, you are right I forgot the await.

File: README.md (hunk 1 above)
Comment (GoorLavi) [reply README.md]: ill change it but it's the same

**Pull request merged**